src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -12,6 +12,7 @@
 using System.Threading;
```

Comment:
@SteveL-MSFT, your last commit had 2 failures in `PowerShell-CI-windows`
Certificate Provider tests.Error occurred in Describe block
```powershell
Expected 2, but got 0.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\Assertions\Should.ps1:186 char:9
```
CmsMessage cmdlets thorough tests.Error occurred in Describe block
```powershell
Expected 2, but got 0.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.1\win7-x64\publish\Modules\Pester\4.8.0\Functions\Assertions\Should.ps1:186 char:9
```